app-backup/backuppc/backuppc-4.4.0.ebuild

```
@@ -0,0 +1,187 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_P="BackupPC-${PV}"
+
+inherit depend.apache systemd
+
+DESCRIPTION="High-performance backups to a server's disk"
+HOMEPAGE="http://backuppc.sourceforge.net/"
+SRC_URI="https://github.com/${PN}/${PN}/releases/download/${PV}/${MY_P}.tar.gz"
+S="${WORKDIR}/${MY_P}"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="rss samba"
+
+DEPEND="
+	acct-group/backuppc
+	acct-user/backuppc
+	app-admin/apache-tools
+	app-admin/makepasswd
+	dev-lang/perl
+"
+
+# The CGI modules are handled in ${RDEPEND}.
+APACHE_MODULES="apache2_modules_alias," # RedirectMatch
+APACHE_MODULES+="apache2_modules_authn_core," # AuthType
+APACHE_MODULES+="apache2_modules_authz_core," # Require
+APACHE_MODULES+="apache2_modules_authz_host," # Require host
+APACHE_MODULES+="apache2_modules_authz_user" # Require valid-user
+
+# Older versions of mod_perl think they're compatible with apache-2.4,
+# so we require the new one explicitly.
+RDEPEND="
+	${DEPEND}
+	app-arch/par2cmdline
+	dev-perl/Archive-Zip
+	dev-perl/CGI
+	dev-perl/File-RsyncP
+	dev-perl/libwww-perl
+	dev-perl/BackupPC-XS
+	net-misc/rsync-bpc
+	virtual/mta
+	virtual/perl-IO-Compress
+	www-apache/mod_perl
+	www-apache/mpm_itk
+	|| (
+		>=www-servers/apache-2.4[${APACHE_MODULES},apache2_modules_cgi]
+		>=www-servers/apache-2.4[${APACHE_MODULES},apache2_modules_cgid]
+	)
+	rss? ( dev-perl/XML-RSS )
+	samba? ( net-fs/samba )"
+
+CGIDIR="/usr/lib/backuppc/htdocs"
+CONFDIR="/etc/BackupPC"
+DATADIR="/var/lib/backuppc"
+DOCDIR="/usr/share/doc/${PF}"
+LOGDIR="/var/log/BackupPC"
+need_apache2_4
+
+src_prepare() {
+	default
+
+	# Fix initscript
+	#sed -e 's/runscript/openrc-run/g' -i init.d/src/gentoo-backuppc || die
+
+	# Fix docs location using the marker that we've patched in.
+	sed "s+__DOCDIR__+${DOCDIR}+" -i "lib/BackupPC/CGI/View.pm" || die
```

I wouldn't bother quoting the path to the file, as it can't contain spaces, but it's somewhat personal preference anyway:
```suggestion
	sed "s+__DOCDIR__+${DOCDIR}+" -i lib/BackupPC/CGI/View.pm || die
```